resources/js/login/login.js

```diff
@@ -368,7 +368,7 @@ const ThirdPartyIdentityProviders = ({ thirdPartyProviders, formAction, disableI
                             target="_self"
                             title={`Sign In with ${provider.label}`}
                             href={`${formAction}/${provider.name}`}>
-                            {provider.label}
+                            {provider.label === "facebook" ? "Log in with " : "Sign in with "}{provider.label}
```

Comment:
this is not correct
if u check the source of this page
[SpeakersSummitRegistrationPromoCode](view-source:https://idp.dev.fnopen.com/auth/login)
labels are
 thirdPartyProviders: [
                                {label: "Facebook", name:"facebook"},
                                 {label: "Apple", name:"apple"},
                                 {label: "Linkedin", name:"linkedin"},
                                 {label: "Okta", name:"okta"},
                             ],

Reply:
also lets create a helper method and give it a meaningful name so you could tell the rationale of this just reading the code

Reply:
@smarcet I am going to put the helper method in `resources/js/utils.js` if there's no objection

Reply:
sure also check that we need to same for title of the anchor element
regards